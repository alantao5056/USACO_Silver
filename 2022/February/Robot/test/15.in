40
2174054 -9156279
-462530838 772820811
-761793545 -905230760
460356784 -763664532
457299814 -540201167
234455435 -509070712
618289309 -29172772
50034857 -195174669
-620463363 38329051
-427392615 -426195018
624468730 497906887
-622294676 -507063166
-307145004 226550054
425218561 435351297
762174804 60611716
735051694 17927019
-969908727 922005298
813093884 122067922
763967599 896074481
-47860803 186018390
363799219 376587809
-715804786 401091303
-790537079 -833784111
-218327380 -644169670
-74886225 480757682
720226503 -920557326
-737225748 -8770740
788363025 842940390
-328405233 647010942
-232281381 499914433
-810919830 -131224201
77060279 -489913961
-722400557 929713605
220501434 635013391
713630732 -391935024
-760000750 -69767995
-365973273 -367431530
967734673 -912849019
330579287 -656167221
309319058 -235706333
-455125760 531044888
