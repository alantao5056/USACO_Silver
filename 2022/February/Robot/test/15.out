0
11
0
0
0
495
0
0
0
5940
0
0
0
27720
0
0
0
58212
0
0
0
58212
0
0
0
27720
0
0
0
5940
0
0
0
495
0
0
0
11
0
0
